core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts

```diff
@@ -638,14 +618,20 @@ export class ServiceMetricsComponent implements OnInit, AfterViewInit {
       }
     })
   }
+  setEndTime() {
```

Comment:
This does it, but a cleaner implementation would be:
this.endTime = moment().toISOString();
and pass the endTime into setStartTime(this.endTime);
and in setStartTime instead of calling moment() again just use the endTime there.

Reply:
@swati-sh Also, make sure the same issue doesn't exist at the environment level as well.

Reply:
Then how the substraction of 24 Hours/1 day/30 days and so on will be supposed to done?

Reply:
End time = moment()
Start time = moment().substract(1,"days)
Reference: https://momentjscom.readthedocs.io/en/latest/moment/03-manipulating/02-subtract/

Ping me if you need further clarification. @SauarvD - fyi

Reply:
addressed

Reply:
@swati-sh have u applied this to stack and tested it? Earlier you had `startTime` and `endTime` as moment().toISOString() - but the same is not being done? Make sure it works as expected. The values of startTime and endTime are consistent now.

Reply:
yes, It is not applied to stack yet.